src/sentry/replays/blueprints/api.md

```diff
@@ -608,3 +608,107 @@ Retrieve a collection of click events associated with a replay.
     ]
   }
   ```
+
+## Replays Viewed [/organizations/<organization_slug>/replays-viewed/]
+
+- Parameters
+
+  - project (optional, string)
+  - id (required, array[string]) - The id of the replay to retrieve. Can be specified multiple times.
+  - per_page (optional, number) - The per page limit should match the number of ids requested.
+    Default: 10
+  - offset (optional, number)
+    Default: 0
+
+### Browse Replays Viewed [GET]
+
+Returns a collection of replays and their viewed state. The user is targetted based on the authorization context. Replay-ids are specified in the URL parameters.
+
+**Attributes**
+
+| Column   | Type   | Description                                              |
+| -------- | ------ | -------------------------------------------------------- |
+| id       | string | A replay-id.                                             |
+| has_seen | bool   | Returns true if the authorized user has seen the replay. |
+
+- Request
+
+  - Headers
+
+    Cookie: \_ga=GA1.2.17576183...
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "id": "81220fbf68494b30a5129ceb4cc5937d",
+        "has_seen": true
+      }
+    ]
+  }
+  ```
+
+## Replay Viewed [/organizations/<organization_slug>/replays/<replay_id>/viewed/]
```

Comment:
Can we title this to specify more clearly that it's an endpoint for fetching the replay _viewers_?

Maybe `Replay Viewers` or `Replay Viewed By`